ansible_collections/arista/avd/roles/eos_designs/templates/facts/topology/p2p-uplinks.j2

```diff
@@ -25,6 +25,10 @@ topology:
 {%             if switch.uplink_ptp.enable is arista.avd.defined %}
       ptp: {{ switch.uplink_ptp.enable }}
 {%             endif %}
+{%             if switch.uplink_macsec.profile is arista.avd.defined %}
+      mac_security:
```

Comment:
you could just do `mac_security: {{ switch.uplink_macsec }}` so any other future options are caried along.
Adjust the `if` above to just test for `uplink_macsec is arista.avd.defined`